### PR TITLE
Apollo playlist ui update

### PR DESCRIPTION
**Pull request by marcelinkaaa**
@gubatron worked well. Thanks. 
I separated all my other list ui updates, so this branch only focuses on updating playlists. 
screenshot_20170619-100819

### CODE REVIEW

**Comment by aldenml** [comment]
codacy?

Is this good now?

File: android/apollo/src/com/andrew/apollo/adapters/PlaylistAdapter.java [hunk 1]
```diff
@@ -12,16 +12,22 @@
 package com.andrew.apollo.adapters;
 
 import android.content.Context;
+import android.graphics.Typeface;
+import android.util.Log;
```
Comment (aldenml) [inline android/apollo/src/com/andrew/apollo/adapters/PlaylistAdapter.java]: can you remove this?

**Comment by marcelinkaaa** [comment]
@aldenml will fix unused imports... there is one more issue @gubatron knows about, assigned him to review. He told me to reopen the pull so he can take a look.

**Comment by aldenml** [comment]
OK

**Comment by gubatron** [comment]
just rebased this to the top to start working on it

**Comment by gubatron** [comment]
@marcelinkaaa just fixed the double rendering bug when you scrolled issue.
But I've the following problem on my emulator, I've to test on my device, the title of the first item is not displaying, wonder if this happened to you, or maybe I just introduced that bug with my null check on convertView
screen shot 2017-06-19 at 2 33 38 pm

**Comment by gubatron** [comment]
i see the text on the samsung, gonna go ahead and merge this

**Comment by marcelinkaaa** [comment]
Actually I could see the same thing when I was doing that same check. So happened on mine. I will test again now.

**Pull request merged**